### DISCUSSION TITLE
HydroDyn: 2ND-ORDER FLOATING PLATFORM FORCES(DiffQTF and SumQTF)

### DISCUSSION THREAD

**Discussion by LeoHongLiZuo** [OpenFAST/openfast#2179] [Q&A]
Dear guys,
Does anyone have the .12d and .12s WAMIT file of the LIFES50+ 10MW OO-Star Semi floating substructure?
![图片](https://github.com/OpenFAST/openfast/assets/108517989/3978ac48-4cb1-4718-9f55-66af6df0a127)
My process of generating this file was not satisfactory.
Or does anyone know what would be a good replacement for these .12d and .12s file
Your sincerely,
Leo

**Comment by Liuzuop** [comment]
Dear Leo,  

Hello, could you please share the HydroD modeling of the OO-star model in OpenFast?
